internal/api/controllers/public/conversions.go

```diff
@@ -71,6 +73,12 @@ func dbRuntoApiRun(r *dbModel.Run, fields []string) *Run {
 		case fieldStatus:
 			value := RunStatus(r.Status)
 			run.Status = &value
+		case fieldName:
+			value := PlaybookName(*r.PlaybookName)
```

Comment:
What happens if r.PlaybookName is nil?

Reply:
Good catch! It should now show up as `"name": ""` for `r.PlaybookName = nil`

Reply:
@tahmidefaz for the event interface we've already starting using `"name": null` in such case. Let's do that here also for consisntency.

Reply:
Changed this to omit the `name` field when `nil` as per our discussion. :+1: